Array/ArrayHomework1.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+$calculate = array(5, 12, 17, 9, 3);
+
+$avg = 0;
+$sum = 0;
+$min = $calculate[0];
+$max = $calculate[0];
+
+for ($i = 0; $i < 5; $i++) {
+    $sum += $calculate[$i];
+    $avg = $sum / 5;
```

Comment:
You only need to calculate average one time, don't need to let it inside the `loop`